lib/puppet/provider/scheduled_task/taskscheduler_api2.rb

```diff
@@ -164,6 +164,11 @@ def enabled=(value)
     end
   end
 
+  def compatibility=(value)
+    # It tries to send :"1" as the value. This works but is clearly wrong.
+    task.compatibility = value.to_s.to_i
```

Comment:
Can we trace this back a bit further before proceeding with this?

Reply:
Definitely, I meant to ask earlier if you had any idea why it might give that value. Is it possibly because of the newvalue in the type declaration?